eZ/Publish/Core/IO/TolerantIOService.php

```diff
@@ -73,7 +73,7 @@ public function loadBinaryFile($binaryFileId)
     {
         $this->checkBinaryFileId($binaryFileId);
 
-        if ($binaryFileId[0] === '/') {
+        if ($binaryFileId[0] === '/' || (PHP_OS === 'WINNT' && $binaryFileId[1] === ':')) {
```

Comment:
This test is supposed to be for public http absolute path, iirc.

After thinking a bit, what might be happening is that tests are using an absolute path. It would work on linux, since both public http path and system path start with a `/`, but not on windows.

Could you investigate in that direction @alongosz ?